velox/functions/lib/window/Ntile.cpp

```diff
@@ -19,30 +19,32 @@
 #include "velox/expression/FunctionSignature.h"
 #include "velox/vector/FlatVector.h"
 
-namespace facebook::velox::window::prestosql {
+namespace facebook::velox::functions::window {
 
 namespace {
 
+template <typename TResult>
```

Comment:
Would you document that TResult can be either int32_t or int64_t? It would be nice to add a static assert for that as well.